reference/tax_provider.yml

```diff
@@ -329,7 +339,13 @@ paths:
   /commit:
     post:
       summary: Commit Tax Quote
-      description: 'Submit the quote request to be persisted by the enabled third-party tax provider. A commit operation is intended to be submitted once only, when the Order has been confirmed and paid.'
+      description: |-
+        Submit the quote request to be persisted by the enabled third-party tax provider. A commit operation is intended to be submitted once only, when the Order has been confirmed and paid.
+
+        > Server URL
+        > - For supporting tax providers, the server url contains the tax provider's profile field; for example, `your_profile.example.com`.
+        > - The Send a Test Request feature is not currently supported for this endpoint.
```

Comment:
```suggestion
        > - The Send a Test Request feature is not currently supported for this endpoint.
```
```suggestion
        > - The Try it feature is not currently supported for this endpoint.
```

Reply:
I see that we still say "Send a Test Request" on the [Storefront API pages](https://developer.bigcommerce.com/docs/rest-storefront/carts#get-a-cart), so if we plan on changing all of them to "Try It", we should do it on a separate ticket / PR (keep things consistent across all docs)